### PR TITLE
2022 Honda Civic DBC

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Three messages (so far) are different on the 2022 Civic than previous Bosch vehicles. Update generator files accordingly.

### CODE REVIEW

File: generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc [hunk 1]
```diff
@@ -18,11 +18,47 @@ BO_ 432 STANDSTILL: 7 VSA
  SG_ COUNTER : 53|2@0+ (1,0) [0|3] "" EON
  SG_ CHECKSUM : 51|4@0+ (1,0) [0|15] "" EON
 
+BO_ 479 ACC_CONTROL: 8 EON
```
Comment (adeebshihadeh) [inline generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc:21]: don’t copy these into individual car DBCs. shared definitions should only be defined once

Comment (vanillagorillaa) [reply generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc:21]: Don't think this is possible. On Bosch vehicles ACC_CONTROL is on ID 479 while the new 2022 Civic is 477. The LKAS_HUD message on Bosch vehicles is 5 bytes while the 2022 Civic is 8 bytes (so checksum and counter messages move).

Comment (adeebshihadeh) [reply generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc:21]: there’s two options here: give the new one a different name or split up the shared DBC into two different files. not sure which I prefer yet, depends on whether the new message is exactly the same but on a different address

Comment (ZgLijian) [reply generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc:21]: 中国的思域 同款车型 INTEGRA 是否支持

Comment (ZgLijian) [reply generator/honda/honda_civic_sedan_16_diesel_2019_can.dbc:21]: Does China's civic intergra support the same model

**Comment by ZgLijian** [comment]
Does China's civic intergra support the same model

**Pull request closed without merging**